commit 67e7b0bf41b6f2cafd3078542c29612f34565fd7
Author: Anjandev Momi <anjan@momi.ca>
Date:   Sat Dec 10 12:22:25 2022 -0800

    support pulseaudio if installed
    
    SDM845 devices need pulseaudio over pipewire for calls to work.
    
    https://gitlab.com/postmarketOS/pmaports/-/issues/1821

diff --git a/configs/default_hooks/sxmo_hook_start.sh b/configs/default_hooks/sxmo_hook_start.sh
index 94255f9..4287886 100755
--- a/configs/default_hooks/sxmo_hook_start.sh
+++ b/configs/default_hooks/sxmo_hook_start.sh
@@ -21,9 +21,14 @@ while ! superctl status > /dev/null 2>&1; do
 done
 
 # Load our sound daemons
-# pipewire-pulse will start pipewire
-superctl start pipewire-pulse
-superctl start wireplumber
+
+if [ "$(command -v pulseaudio)" ]; then
+	superctl start pulseaudio
+elif [ "$(command -v pipewire)" ]; then
+	# pipewire-pulse will start pipewire
+	superctl start pipewire-pulse
+	superctl start wireplumber
+fi
 
 # mako/dunst are required for warnings.
 # load some other little things here too.
diff --git a/configs/superd/services/pulseaudio.service b/configs/superd/services/pulseaudio.service
new file mode 100644
index 0000000..6f65feb
--- /dev/null
+++ b/configs/superd/services/pulseaudio.service
@@ -0,0 +1,7 @@
+[Unit]
+Description=general purpose sound server intended to run as a middleware between your applications and your hardware devices
+
+[Service]
+Type=simple
+Restart=always
+ExecStart=/usr/bin/pulseaudio
diff --git a/scripts/core/sxmo_version.sh b/scripts/core/sxmo_version.sh
index 90a28e9..77640e1 100755
--- a/scripts/core/sxmo_version.sh
+++ b/scripts/core/sxmo_version.sh
@@ -28,6 +28,7 @@ esac
 
 printf "superd "
 /usr/bin/superctl --version
+pactl info
 "$KEYBOARD" -v
 /usr/bin/mmcli --version | head -n 1
 uname -m
